src/commonMain/kotlin/teksturepako/pakku/api/projects/ProjectFile.kt

```diff
@@ -27,6 +29,7 @@ data class ProjectFile(
     val hashes: MutableMap<String, String>? = null,
     @SerialName("required_dependencies") val requiredDependencies: MutableSet<String>? = null,
     val size: Int = 0,
+    val dataPublished: @Serializable(with = InstantIso8601Serializer::class) Instant = Instant.MIN,
```

Comment:
This is not backwards compatible. It should be nullable and null by default.

Reply:
It is backwards compatible, since it has a default value. ok

Reply:
Ah, sorry I misread that.

Reply:
huh. Should I modify it to nullable?

Reply:
No, I mean it was my mistake that I didn't notice the default `Instant.MIN` value,